src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java

```diff
@@ -139,41 +135,48 @@ private TableDescription getTableDescriptionFromCache(String tableName) throws R
         }
     }
 
-    private TableDescription getTableDescriptionNoCache(String tableName) {
+    private MtTableDescription getTableDescriptionNoCache(String tableName) {
+        // first get description for table name at the current context
+        Optional<MtTableDescription> tableDescription = getTableDescriptionFromDb(tableName);
```

Comment:
Do we need/want to namespace virtual mt tables and virtual tables? If not, would tenant-level tables shadow mt tables?

Reply:
To me it seems perhaps more confusing for a user to be able to define both a virtual mt table and a virtual tenant-level table with the same name, so I went with making them share the same namespace. What do you mean by tenant-level tables shadowing mt tables?